frontend/shared-components/text-input/text-input.tsx

```diff
@@ -0,0 +1,48 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+import React, { FormEvent, useState } from 'react'
+import styles from './text-input.module.css'
+import SyntaxHelper from '../../helpers/syntax-helper'
+import { InputNames, inputValidation } from '../../models' 
+
+interface TextInputProps { inputName: InputNames, isSubmitted: boolean, isPasswordShown?: boolean  }
```

Comment:
duplicate component?

Reply:
No? the wrapping directory holds the tsx & css files together if that's what you're referring to?